### ISSUE TITLE
[BUG] TypedDict "ClientErrorResponseTypeDef" has no key 'Error'

### ISSUE THREAD

**Issue by pgrz** [vemel/mypy_boto3_builder#85]
**Describe the bug**
The stub for 'botocore.exceptions.ClientError' gets lowercase 'error' though the [botocore](https://github.com/boto/botocore/blob/develop/botocore/exceptions.py#L425) code uses capitalized 'Error'

**To Reproduce**
1. Install 'botocore-stubs==1.20.91' and 'botocore==1.20.91'
2.  Run mypy on the following code sample

```python
from botocore.exceptions import ClientError


def print_response_code(exception: ClientError):
    print(exception.response["Error"]["Code"])
```
3. Actual mypy output

```
error: TypedDict "ClientErrorResponseTypeDef" has no key 'Error'
note: Did you mean "error"?
```

**Expected mypy-output**

```
Success: no issues found in 1 source file
```

**Additional context**
Installed on MacOS in a fresh pyenv, run with mypy==0.790

**Comment by vemel** [comment]
Thank you for the report. This bug is already fixed in master, I am going to release it today.

**Comment by vemel** [comment]
Fix included to `boto3-stubs 1.17.92.post1`. Please let me know if it works as expected.

**Comment by pgrz** [comment]
It works, thank you
